src/Elcodi/Admin/CoreBundle/Resources/translations/messages.ca.yml

```diff
@@ -1,4 +1,5 @@
 admin:
+    name: Administració
```

Comment:
shouldn't this have been `administrador` https://translate.google.com/#auto/ca/Administrator ?